src/Foundation.Find.Cms/Config/FacetFilterConfigurationItem.cs

```diff
@@ -0,0 +1,163 @@
+﻿using EPiServer.DataAnnotations;
+using EPiServer.Globalization;
+using Foundation.Find.Cms.Facets;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done